applications/plugins/Geomagic/src/Geomagic/GeomagicDriver.cpp

```diff
@@ -334,11 +334,7 @@ void GeomagicDriver::initDevice()
     }
     
     // 2.6- Need to wait several ms for the scheduler to be well launched and retrieving correct device information before updating information on the SOFA side.
-#ifndef WIN32
-        usleep(42);
-#else
-        Sleep(42);
-#endif
+    std::this_thread::sleep_for(42);
```

Comment:
You are sure that std::this_thread::sleep_for(42) will sleep for 42 ms?

Reply:
in fact this will sleep at least for 43ms... ;)  I discover that sleep_for method take more than 1 ms to act....